internal/mode/static/nginx/config/generator.go

```diff
@@ -43,11 +43,13 @@ type Generator interface {
 //
 // It also expects that the main NGINX configuration file nginx.conf is located in configFolder and nginx.conf
 // includes (https://nginx.org/en/docs/ngx_core_module.html#include) the files from httpFolder.
-type GeneratorImpl struct{}
+type GeneratorImpl struct {
+	plus bool
+}
 
 // NewGeneratorImpl creates a new GeneratorImpl.
-func NewGeneratorImpl() GeneratorImpl {
-	return GeneratorImpl{}
+func NewGeneratorImpl(plus bool) GeneratorImpl {
```

Comment:
@kate-osborn Figured this one was okay since it's a constructor. What do you think?